test/specs/happy.js

```diff
@@ -639,20 +639,20 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
       }
 
       it('should navigate to the second-last step of the flow and then go back to the beginning', async () => {
-        const copy = common.copy(lang)
+        const copy = screen.copy(lang)
         goToPassportUploadScreen(`?language=${lang}&async=false&useWebcam=false`)
         uploadFileAndClickConfirmButton('passport.jpg')
         documentUpload.getUploadInput()
         documentUpload.upload('face.jpeg')
-        common.clickBackArrow()
+        screen.clickBackArrow()
```

Comment:
hm, it would make more sense to call `clickBackArrow` on the screen we assume the user is in

Reply:
In that case 

```
async clickBackArrow() {
    this.backArrow.click()
}
```
would have to be in every page object class. If Screen class is not for the selectors that repeat across the different screens it makes sense to move it. 

So for now, Screen would only be for:

```copy(lang) {
    return locale(lang)}

Reply:
>  If Screen class is not for the selectors that repeat across the different screens it makes sense to move it.
I don't follow this statement.

all page objects that you press back on are screens, so they all should inherit from the super call screen, which should have the method to click back

Reply:
screen shouldn't really be treated as instance, but more like an abstract class, which is to say a class that a concrete will inherit from.

Reply:
Thanks for explanation. This is now done in 38cf58a